frontend/src/mixin.js

```diff
@@ -19,26 +19,26 @@ export default Vue.mixin({
     startToday() {
       return startOfToday()
     },
-      secondsHumanize (val) {
-        const t2 = addSeconds(new Date(0), val)
-          if (val >= 60) {
-              let minword = "minute"
-              if (val >= 120) {
-                  minword = "minutes"
-              }
-              return format(t2, "m '"+minword+"' s 'seconds'")
-          }
-        return format(t2, "s 'seconds'")
-      },
+    secondsHumanize (val) {
+      const t2 = addSeconds(new Date(0), val)
```

Comment:
Same like in the other PR. The reformatting can be done in a second commit.

Reply:
Fixed